clayui.com/src/components/clay/Editor.js

```diff
@@ -109,10 +114,27 @@ const Editor = ({code, disabled = false, imports, preview = true, scope}) => {
 					<LiveError />
 				</div>
 			)}
+
 			<div className="gatsby-highlight">
 				<div style={{padding: '10px'}}>
-					{imports && <LiveEditor disabled value={imports} />}
-					<LiveEditor />
+					<ClayButtonWithIcon
```

Comment:
We might as well also add the copy button here too to keep consistent with the css code snippets. What do you think?

Reply:
So, adding the `copy` button here would go along with the goal of having a single implementation of collapsing. I didn't know how all of these implementations were connected and there isn't a `copy` button on the React implementation of code examples.

What I'm trying to say is that, yes, I'll move the copy button here which will add the ability to copy code from all examples, not just markup/css ones.